ffi/src/value/kvp.rs

```diff
@@ -61,3 +61,23 @@ impl<'a> api::KeyValuePair for &KeyValuePair<'a> {
         (*self).into_batch()
     }
 }
+
+/// Owned version of `KeyValuePair`, returned to ffi callers.
+///
+/// C callers must free this memory using the respective FFI function for the
+/// concrete type (but not using the `free` function from the C standard library).
```

Comment:
This is still a bit confusing. This is a comment for `OwnedKeyValuePair` which is a concrete type. What is the "respective FFI function for the concrete type" if the concrete type is `OwnedKeyValuePair`?